recipes/shapelysmooth/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "shapelysmooth" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shapelysmooth-{{ version }}.tar.gz
+  sha256: 02141a11f8f615b418a854475292c107b3e2c6bc5f56e9b683eea9f63ac288a3
+
+build:
+  skip: True  # [win or osx]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pybind11
+    - pip
+  run:
+    - python
+    - libgcc-ng
+    - libstdcxx-ng
```

Maybe enable windows/mac above and do:
```suggestion
    - libgcc-ng  # [linux]
    - libstdcxx-ng  # [linux]
```